A32NX/html_ui/Pages/A32NX_Core/A32NX_Pilots.js

```diff
@@ -0,0 +1,9 @@
+class A32NX_Pilots {
+    init() {
+        const storedPilotVis = parseInt(NXDataStore.get("CONFIG_PILOT_VISIBILITY", "0"));
+        const storedCoPilotVis = parseInt(NXDataStore.get("CONFIG_COPILOT_VISIBILITY", "0"));
+
+        SimVar.SetSimVarValue("L:A32NX_VIS_PILOT_0", "Number", storedPilotVis);
+        SimVar.SetSimVarValue("L:A32NX_VIS_PILOT_1", "Number", storedCoPilotVis);
+    }
```

Comment:
```suggestion
    }
    update(_deltaTime, _core) {
    }
```

Reply:
you should add update even not use that.
also if you add init code, you can check that js is loaded